core/domain/html_validation_service.py

```diff
@@ -877,15 +880,40 @@ def get_filename_with_dimensions(old_filename, exp_id):
     return new_filename
 
 
-def get_invalid_svg_tags(svg_string):
-    """Returns a set of all invalid tags for the provided SVG.
+def get_invalid_svg_tags_and_attrs(svg_string):
+    """Returns a set of all invalid tags and attributes for the provided SVG.
 
     Args:
         svg_string: str. The SVG string.
 
     Returns:
-        set. A set of invalid tags.
+        tuple. A 2-tuple, the first element of which is a list of invalid tags,
```

Comment:
`tuple(list(str))` perhaps?

Though see previous comment about validity of attrs being a tag-specific concept.